src/vsphere_cpi/spec/unit/cloud/vsphere/resources/network_spec.rb

```diff
@@ -0,0 +1,216 @@
+require 'spec_helper'
+require 'rspec/its'
+
+describe VSphereCloud::Resources::Network do
+  subject do
+    described_class.new('foo_net', mob, client, )
+  end
+
+  let(:client) { instance_double('VSphereCloud::VCenterClient', cloud_searcher: cloud_searcher) }
+  let(:cloud_searcher) { instance_double('VSphereCloud::CloudSearcher') }
+  let(:vim_network_name) { 'fake-network' }
+  let(:mob) do
+    instance_double('VimSdk::Vim::Network', to_s: 'mob_as_string', name: vim_network_name)
+  end
+
+  describe '#vim_name' do
+    it 'should return network with vim name as fake-network' do
+      expect(subject.vim_name).to eq(vim_network_name)
+    end
+  end
+
+  describe '#nic_backing' do
+    it 'returns backing of type plain NetworkBackingInfo ' do
+      backing_info = subject.nic_backing
+      expect(backing_info).to be_a(VimSdk::Vim::Vm::Device::VirtualEthernetCard::NetworkBackingInfo)
```

Comment:
`be_an_instance_of` to make sure that it's not a subclass.

Reply:
done